frontend/src/components/Statistics/Statistics.tsx

```diff
@@ -1,35 +1,41 @@
 import React, { useEffect, useState } from 'react';
 import { Col, Container, Row } from 'react-bootstrap';
-import { Button as MuiButton, TextField } from '@material-ui/core';
-import Autocomplete from '@material-ui/lab/Autocomplete';
+import { Button as MuiButton } from '@material-ui/core';
 import './Statistics.scss';
 import { CanvasJSChart } from '../../libs/canvasjs.react';
 import { DjangoBackend } from '../../api/DjangoBackend';
-import { Event, MenuItem, Order } from '../../@types';
+import { MenuItem, Order } from '../../@types';
+import { DateTimePicker, MuiPickersUtilsProvider } from '@material-ui/pickers';
+import DateFnsUtils from '@date-io/date-fns';
+import { useSnackbar } from 'notistack';
 
 function Statistics() {
-    const [events, setEvents] = useState<Event[]>([])
+    const [startDate, setStartDate] = useState(new Date(Date.now()-3600*24*1000))
+    const [endDate, setEndDate] = useState(new Date())
     const [menuItems, setMenuItems] = useState<MenuItem[]>([])
     const [isLoadingData, setIsLoadingData] = useState(false)
-    const [selectedEvent, setSelectedEvent] = useState<Event | null>(null)
     const [chartOptions, setChartOptions] = useState({})
+    const { enqueueSnackbar } = useSnackbar()
+
 
     useEffect(() => {
-        Promise.all([DjangoBackend.get<Event[]>('/api/events/'), DjangoBackend.get<MenuItem[]>('/api/menu_items/')])
-            .then(([events, menuItems]) => {
-                setEvents(events.data)
+        Promise.all([DjangoBackend.get<MenuItem[]>('/api/menu_items/')])
+            .then(([menuItems]) => {
                 setMenuItems(menuItems.data)
             })
             .catch(reason => console.log(reason.response))
     }, [])
 
-    function onEventSubmit(event: React.FormEvent<HTMLFormElement>) {
+    function onDateSubmit(event: React.FormEvent<HTMLFormElement>) {
         event.preventDefault()
         setIsLoadingData(true)
-        DjangoBackend.get<Order[]>(`/api/orders_with_order_items/?event=${selectedEvent?.id}`)
+        var hasValue = false;
```

Comment:
Use `let` instead of `var`
```suggestion
        let hasValue = false;
```